swagger/v3/component_schemas/ECFSchoolsFilter.yml

```diff
@@ -2,6 +2,10 @@ description: "Filter schools to return more specific results"
 type: object
 properties:
   cohort:
-    description: "Return schools within the specified cohort"
+    description: "Return schools within the specified cohort(s). This is a comma delimited string of years."
```

Comment:
Ah we shouldn't allow multiple cohorts for schools, as each school will have different data per cohort (the in partnership field) so it only can be one